spring-cloud-huawei-servicecomb-discovery/src/main/java/com/huaweicloud/servicecomb/discovery/discovery/MicroserviceHandler.java

```diff
@@ -39,9 +41,15 @@
 
   private static List<ServiceInstance> instanceList = Collections.emptyList();
 
+  public static Map<String, String> serviceRevision = new ConcurrentHashMap<>();
```

Comment:
Define a setter is better, e.g. addServiceReivison(String, String). Or must difine it to be final .

Reply:
change it to final